### PR TITLE
feat: Add Node script which stops at END_URL

### PR DESCRIPTION
**Pull request by lukehesluke**

### CODE REVIEW

File: walkRpde.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ */
+async function walkRpde(startUrl, options) {
+  let prevNextUrl;
+  let nextUrl = startUrl;
+  let page = 0;
+  while (doContinueIterating({ nextUrl, prevNextUrl, endUrl: options.endUrl })) {
+    const { data: pageJson } = await axios.get(nextUrl);
+    const filePath = path.join(__dirname, `rpde-${page}.json`)
+    await writeFile(filePath, JSON.stringify(pageJson, null, 2));
+    nextUrl = pageJson.next;
```
Comment (house92) [inline walkRpde.js]:
```suggestion
    prevNextUrl = nextUrl;
    nextUrl = pageJson.next;
```

File: walkRpde.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+/**
+ * @param {string} startUrl
+ * @param {object} options
+ * @param {string} [options.endUrl]
+ */
+async function walkRpde(startUrl, options) {
+  let prevNextUrl;
+  let nextUrl = startUrl;
+  let page = 0;
```
Comment (house92) [inline walkRpde.js]: I think in order to preserve file order we should stick with the existing behaviour of at least three digits in the file name. Maybe even bump that up to four, given how large GLL is

File: walkRpde.js (hunk 1 above)
Comment (lukehesluke) [reply walkRpde.js]: Oops!!

Comment (lukehesluke) [reply walkRpde.js]: This has been done now

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  "name": "rpde-scripts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "walkRpde.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/imin-ltd/rpde-scripts.git"
```
Comment (civsiv) [inline package.json]: what's the `git+` bit?

Comment (lukehesluke) [reply package.json]: https://askubuntu.com/a/858728

**Pull request merged**